app/core/services/document_manipulation/document_manipulation.py

```diff
@@ -144,7 +143,7 @@ async def convert_pdf_to(
         first_page_number=first_page_number,
         last_page_number=last_page_number,
     )
-    return await _convert_with_libre(
+    return await convert_file_to(
```

Comment:
I prefer having a single function as an entry point to call LibreOffice and UnoServer interactions after all the problems it gave us, but it could be changed if it seem useless

Reply:
It makes sense, I would add a small comment on that part! :wink: